transport/tchannel/outbound.go

```diff
@@ -67,6 +67,11 @@ func (o *Outbound) WithTracer(tracer opentracing.Tracer) *Outbound {
 	return o
 }
 
+// Transports returns the underlying TChannel Transport for this outbound.
+func (o *Outbound) Transports() []transport.Transport {
+	return []transport.Transport{}
```

Comment:
Why instantiate here? Shouldn't the transport pass an instance of itself into the outbound? Wouldnt this just return that?